samples/ConPTY/EchoCon/EchoCon/EchoCon.cpp

```diff
@@ -1,64 +1,53 @@
-// EchoCon.cpp : Entry point for the EchoCon sample application.
-//
+// EchoCon.cpp : Entry point for the EchoCon Pseudo-Consle sample application.
+// Copyright � 2018, Microsoft
 
 #include "stdafx.h"
 #include <Windows.h>
-#include <conio.h>
 #include <process.h>
-#include <wchar.h>
 
 // Forward declarations
-void __cdecl PipeListener(LPVOID);
+HRESULT CreateConPTY(HPCON*, HANDLE*, HANDLE*);
```

Comment:
Can you standardize on one or the other? Please?

Reply:
The `PipeListener()` is a thread handler and has to conform to the signature expected by the CRT-safe `_beginThread()`

Reply:
That's entirely immaterial to my comment.
about "ConPTY", _Can you standardize on one or the other, please?_